docs/src/main/paradox/docs/delta/api/projects-api.md

```diff
@@ -190,13 +190,11 @@ Response
 
 Deletes the target project and all its resources. 
 
-Once the request is processed, an asynchronous task gets trigger which deletes the data generated from each resource type (e.g.: the indices generated by the views resources, the binaries generated by the files resources).
+Once the request is processed, an asynchronous task gets triggered and deletes the data generated from each resource type (e.g.: the indices generated by the views resources, the binaries generated by the files resources).
 During this time, no write operation can be performed at the project level nor at the resources level for the target project.
 When the task is finished, the target project can be recreated if desired.
 
-![deletion diagram](assets/projects/projects-delete-diagram.png)
-
-Deletion only works when the configuration flag `app.database.deny-cleanup` is set to `false` and the current project is not referenced somewhere else (e.g.: a `CrossProjectResolver`, an `AggregatedView`, etc.).
+Deletion only works when the configuration flag `app.projects.deletion.enabled` is set to `true` and the current project is not referenced somewhere else (e.g.: a `CrossProjectResolver`, an `AggregatedView`, etc.).
```

Comment:
Perhaps to replace the e.g:

(For example, a CrossProjectResolver or AggregatedView might reference a project)
or
(commonly a CrossProjectResolver or AggregatedView might reference a project)

Reply:
They always do (even if an aggregate view can reference the same project)

Reply:
I meant to communicate that "an Aggregate view might be referencing the target project", just to give a concrete example of what a reference might be.

Perhaps even just saying (e.g.: IN a `CrossProjectResolver`, an `AggregatedView`, etc.).